### DISCUSSION TITLE
What licensing tooling are you using?

### DISCUSSION THREAD

**Discussion by simkimsia** [github-linguist/linguist#5779] [Q&A] [answered]
i noticed how this project has a vendors/licenses/config.yaml i suspect some kind of license tooling is used.

can tell us what tooling is this? cause i am trying to learn how to write and maintain an opensource library that itself is forked from other open source libraries

https://github.com/github/linguist/blob/master/vendor/licenses/config.yml

**Comment by lildude** [comment, marked as answer]
We use [licensed](https://github.com/github/licensed).
